scale/dev/launch_sensor.sh

```diff
@@ -20,12 +20,6 @@ fi
 
 SENSOR_HELM_DEPLOY=false CLUSTER="${namespace}" NAMESPACE_OVERRIDE="${namespace}" "$DIR/../../deploy/k8s/sensor.sh"
 
-# This is purposefully kept as stackrox because this is where central should be run
-if ! kubectl -n stackrox get pvc/stackrox-db > /dev/null; then
```

Comment:
uh, a bit of a nit: let's make this central-db? basically if you run without a pvc the whole thing fails